config/preview.json

```diff
@@ -4,6 +4,6 @@
 		"max_static_paths": 10
 	},
 	"flags": {
-		"enable_hcp_vault_secrets_api_docs_revision": false
+		"enable_hcp_vault_secrets_api_docs_revision": true
```

Comment:
🚧 Will turn these flags `false` before merging. Have set to `true` for dev & review.